frontend/components/WifiDetails.jsx

```diff
@@ -8,7 +8,7 @@ import InfoIcon from '@material-ui/icons/Info';
 
 import { makeStyles } from '@material-ui/core/styles';
 
-const useStyles = makeStyles(theme => ({
+const useStyles = makeStyles((theme) => ({
```

Comment:
Why did you add parentheses here? Please make sure your commits are clean under `yarn format`

Reply:
Oh that might be my prettier extension in vs code using local config rather than project configuration. I did this edit on other machine so that might have added those parentheses. Will resolve it soon.